masterlist.yaml

```diff
@@ -5467,7 +5467,7 @@ plugins:
         condition: 'active("RelightingSkyrim_SSE.esp") and not active("Landscape and Water Fixes - Patch - Relighting Skyrim.esp")'
       - <<: *patchIncluded
         subs: [ 'Static Mesh Improvement Mod' ]
-        condition: 'active("SMIM.*\.esp") and not checksum("meshes/architecture/farmhouse/walkway/walkwaycwall01.nif", E114830E)'
+        condition: 'active("SMIM.*\.esp") and (not checksum("meshes/architecture/farmhouse/walkway/walkwaycwall01.nif", E114830E) and not checksum("meshes/architecture/farmhouse/walkway/walkwaycwall01.nif", 9016DCA9))'
```

Comment:
This is the downside of using CRC for file checks and why I try to avoid them.

Reply:
Definitely, but the patch only consists of meshes. There are also 3 versions of the file (vanilla, SMIM, and Enhanced textures detail (UV-tweaks)), so just checking that the file isn't the one from SMIM isn't enough. I'll make sure to keep an eye on the mod in case it needs updates, though.